ml/rl/workflow/sample_configs/cartpole_discrete_dqn_online.yaml

```diff
@@ -0,0 +1,67 @@
+%YAML 1.2
+---
```

Comment:
Do we need the header?

Reply:
Since we had #!/usr/bin/env python3 headers I thought maybe yaml should have too?